modules/javafx.media/src/main/legal/glib.md

```diff
@@ -28,7 +28,14 @@ code is here:
 
 ```
 
-### LGPL 2.1
+### LICENSE
+```
+There are multiple licenses that apply to different parts of Glib. The
+overarching license is LGPL 2.1, however parts are covered by MIT and
+other parts by BSD-style licenses.
+```
+
+#### LGPL 2.1
```

Comment:
Minor: to match the other sections and also what we did for WebKit, you might change this to `LGPL 2.1 License`